test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -93,6 +100,55 @@ static public void testCreatePoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D), yNew);
 	}
 
+	/*
+	 * Test array of Point2D
+	 */
+	@Test(priority=2)
+	static public void testCreateArrayPoint2D() throws Throwable {
+		MethodHandle withX = generateWither(point2DClass, "x", int.class);
+		MethodHandle withY = generateWither(point2DClass, "y", int.class);
+
+		int x1 = 0xFFEEFFEE;
+		int y1 = 0xAABBAABB;
+		int x1New = 0x11223344;
+		int y1New = 0x99887766;
+		
+		Object point2D_1 = makePoint2D.invoke(x1, y1);
+		
+		assertEquals(getX.invoke(point2D_1), x1);
```

Comment:
dont need to repeat tests in `testCreatePoint2D` just worry about testing arrays